src/main/scala/com/microsoft/hyperspace/index/rules/FilterIndexRule.scala

```diff
@@ -201,11 +146,18 @@ object FilterIndexRule
    * @return top-most index which is expected to maximize performance gain
    *         according to ranking algorithm.
    */
-  private def rank(candidates: Seq[IndexLogEntry]): Option[IndexLogEntry] = {
+  private def rank(
```

Comment:
@sezruby @imback82 Naive question. Looks like `rank` is in `FilterIndexRule.scala` but for `JoinIndexRule`, it is in `/index/rankers/`. I'm guessing this was left as is on purpose?

Reply:
I guess it was because of just a simple function with TODO. I'll move this to `/index/rankers/`.